app/src/components/submissionPage/SubmissionBoxErrorDisplay.tsx

```diff
@@ -1,10 +1,22 @@
 import React from "react";
+import styled from "styled-components";
+
+const ErrorMessage = styled.p`
```

Comment:
`ErrorMessage` is almost identical to `SnapSuccessText` and could be commonised.